test/firewalld-dbus.conf

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-Bus Bus Configuration 1.0//EN" "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
```

Comment:
Wouldn't this file fit better under `testfiles`?

Reply:
yes